tests/test-vm.sh

```diff
@@ -28,7 +29,7 @@ bailout() {
     ps --pid="${QEMU_PID}" -o pid= | grep -q '.' && kill "${QEMU_PID:-}"
   fi
 
-  rm -rf "${TEST_TMPDIR}"
+  sudo rm -rf "${TEST_TMPDIR}"
```

Comment:
hm, only usage of `sudo` in this script here, is that expected?

Reply:
It is, the process of extracting the kernel and initramfs from the RPi image has to be done as root, so the kernel and initramfs end up being owned by root. The script will crash on this `rm -rf` if it isn't run with `sudo`. I did consider just changing the permissions on the kernel and initramfs files instead, but that was more work than this, in a context where I didn't think this had any security implications, so I opted for this intead.

Reply:
I see, thanks for clarifying! :+1: